examples/nucleo_g474re/max31855/main.cpp

```diff
@@ -0,0 +1,99 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2022, Sarah Vilete
+ * Copyright (c) 2022, Rasmus Kleist Hørlyck Sørensen
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <modm/board.hpp>
+#include <modm/processing.hpp>
+#include <modm/driver/temperature/max31855.hpp>
+
+using SpiMaster = modm::platform::SpiMaster2;
+
+using Cs = modm::platform::GpioA10;
+using Miso = modm::platform::GpioB14;
+using Sck = modm::platform::GpioB13;
+
+using namespace Board;
+
+class ThermocoupleThread : public modm::pt::Protothread
+{
+public:
+    ThermocoupleThread() : thermocouple(data) {}
+
+    bool
+    run()
+    {
+        PT_BEGIN();
+
+       	thermocouple.initialize();
```

Comment:
indentation

Reply:
Sorry what do you mean?

Reply:
Could be my browser but on my end it looks like there is one excess space in front of `thermocouple.initialize(); ` in line 35.

Reply:
To me they are at the same level of indentation

Reply:
![indentation](https://user-images.githubusercontent.com/25187160/164512519-cb058ab1-5a37-40cc-8f3f-50dd62f9a2f7.png)

There is a tab character. Also lines 80 and 81 are indented with tabs instead of spaces like the rest of the file.

Reply:
Okay I will change it to spaces